src/main/java/edu/wpi/grip/ui/preview/PreviewsView.java

```diff
@@ -48,24 +59,57 @@ public PreviewsView(EventBus eventBus) {
     }
 
     @Subscribe
+    /**
+     * This function is called when a preview button is pushed/triggered
+     */
     public synchronized void onSocketPreviewChanged(SocketPreviewChangedEvent event) {
-        Platform.runLater(() -> {
```

Comment:
Yea, It leave it behind because its the old one. I saw you took care of it. Thanks!